vimiv/gui/library.py

```diff
@@ -468,17 +478,17 @@ def _add_rows(self, paths: List[str], are_directories: bool = False):
         get_size = files.get_size_directory if are_directories else files.get_size_file
         mark_prefix = api.mark.indicator + " "
         for i, path in enumerate(paths, start=self.rowCount() + 1):
-            name = os.path.basename(path)
+            name = os.path.basename(path[0])
             if are_directories:
                 name = utils.add_html(name + "/", "b")
             if path in api.mark.paths:
                 name = mark_prefix + name
             with contextlib.suppress(FileNotFoundError):  # Deleted in the meantime
-                size = get_size(path)
+                size = get_size(*path)
```

Comment:
This seems to break when we load a changed directory as the working directory handler strips the meta and we end up with a simple list of strings, not a list of tuples.